papers/Vvmax/RLFs.py

```diff
@@ -0,0 +1,125 @@
+import numpy as np
```

Comment:
similar to the data, if any of the code that follows will be useful to the community, it should be in the main repo folder, e.g.

`zdm/vvmax/RLFs.py`

Reply:
I don't think these ones have much value. Mostly, RLFs.py
- keeps constants from the literature, but only for three papers, and only a few constants
- has some specialised routines that integrate the functions over histogram bins
Obviously, they could be useful, but they're also of rather marginal use.